Resources/Prototypes/Entities/Structures/Power/substation.yml

```diff
@@ -42,3 +42,22 @@
     maxChargeRate: 5000
     supplyRampTolerance: 5000
     supplyRampRate: 1000
+  - type: Construction
+    graph: machine
+    node: machine
+  - type: Destructible
+    thresholds:
+    - trigger:
+        !type:DamageTrigger
+        damage: 100
+      behaviors:
+      - !type:ChangeConstructionNodeBehavior
+        node: machineFrame
+      - !type:DoActsBehavior
+        acts: ["Destruction"]
+  - type: Wires
+    BoardName: "Substation"
+    LayoutId: Substation
```

Comment:
The wiring/hacking interface dosent show up but this is just so that it has a maintenance panel to open. Not sure if theres a better way to do that but it seems that its needed to be deconstructable

Reply:
Just tested it again, without the wires component the deconstruction prompt says to use the crowbar but it just doesn't work. Not sure if that's a bug or something

Reply:
looks like its required in the construction graph https://github.com/DubiousDoggo/space-station-14/blob/a0c2c9220aa662e18a4353aa209e4e543bbb05f8/Resources/Prototypes/Recipes/Construction/Graphs/machines/machine.yml#L101